src/status/status-module.ts

```diff
@@ -241,12 +241,11 @@ export default class StatusModule {
   }
 
   public async getStatus(withEcs = false) {
-    const [gitlab, runners, postgresql, screenshotter, systemHook, charlesVersion] = await Promise.all([
+    const [gitlab, runners, postgresql, screenshotter, charlesVersion] = await Promise.all([
       this.getGitlabStatus(),
       this.getRunnersStatus(),
       this.getPostgreSqlStatus(),
       this.getScreenshotterStatus(),
-      this.getSystemHookStatus(),
```

Comment:
Systemhook is not used for anything and this check is creating false positives. We might wish to include it again, but better to avoid noise